flags/settings.py

```diff
@@ -55,10 +55,10 @@ def conditions(self):
         return self.configured_conditions + self.dynamic_conditions
 
     def check_state(self, **kwargs):
-        """ Determine this flag's state based on its conditions """
+        """ Determine this flag's state based on any of its conditions """
         if len(self.conditions) == 0:
             return False
-        return all(fn(v, **kwargs) for c, fn, v, o in self.conditions)
+        return any(fn(v, **kwargs) for c, fn, v, o in self.conditions)
```

Comment:
Does this change negate the need for the line above (`if len(self.conditions) == 0: return False`)?

I recall that getting added because without any conditions you wanted flags to default to false. Is that still the case? So the behavior would be:

- No conditions: `False`
- Some conditions, none met: `False`
- Some conditions, some met: `True`
- Some conditions, all met: `True`

If that behavior is still desired, I think you can remove that line because

```py
>>> all([])
True
>>> any([])
False
```